swagger2.0/oic.wk.rd.swagger.json

```diff
@@ -1,598 +1,476 @@
+            "description" : "Respond with the same schema as publish with the additional \"ins\" Parameter in the Link.\n",
+            "x-example": {
+              "di": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9",
+              "links": [
+                {
+                  "anchor": "ocf://e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9",
+                  "href":   "/myLightSwitch",
+                  "rt":     [ "oic.r.switch.binary" ],
+                  "if":     [ "oic.if.a", "oic.if.baseline" ],
+                  "p":      { "bm": 3 },
+                  "eps": [
+                    { "ep": "coaps://[2001:db8:a::b1d6]:1111", "pri": 2 },
+                    { "ep": "coaps://[2001:db8:a::b1d6]:1122" },
+                    { "ep": "coaps+tcp://[2001:db8:a::123]:2222", "pri": 3 }
+                  ],
+                  "ins":    11235
+                },
+                {
+                  "anchor": "ocf://e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9",
+                  "href":   "/myLightBrightness",
+                  "rt":     ["oic.r.brightness"],
+                  "if":     ["oic.if.a", "oic.if.baseline"],
+                  "p":      {"bm": 3},
+                  "eps": [
+                      {"ep": "coaps://[2001:db8:a::123]:2222"}
+                  ],
+                  "ins":    112358
+                }
+              ],
+              "ttl": 600
+            },
+            "schema": { "$ref": "#/definitions/rdPublish" }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "rdgetinterface" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.baseline"],
+      "description" : "enumdescription"
+    },
+    "rdpostinterface" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.baseline"],
+      "description" : "enumdescription"
+    }
+  },
+  "definitions": {
+    "rdSelection" : {
+      "properties": {
+        "rt" : {
+          "description": "Resource Type of the Resource",
+          "items": {
+            "enum": ["oic.wk.rd"],
+            "type": "string"
+          },
+          "minItems": 1,
+          "maxItems": 1,
+          "uniqueItems": true,
+          "readOnly": true,
+          "type": "array"
+        },
+
+        "n" : {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+
+        "sel" : {
+          "description": "A bias factor calculated by the Resource directory",
+          "maximum": 100,
+          "minimum": 0,
+          "readOnly": true,
+          "type": "integer"
+        },
+
+        "id" : {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+        },
+
+        "if" : {
+          "description": "The interface set supported by this resource",
+          "items": {
+            "enum": [

```

Comment:
Why is the full OCF Interface list included in this enum?

Reply:
As discussed in CTWG this morning and also in Stefano's email; RD needs some additional work.  There will be a further push for this.